webpack.config.js

```diff
@@ -0,0 +1,103 @@
+const path = require('path');
+const webpack = require('webpack');
+const CleanWebpackPlugin = require('clean-webpack-plugin');
+const HtmlWebpackPlugin = require('html-webpack-plugin');
+const ExtractTextWebpackPlugin = require('extract-text-webpack-plugin');
+const BundleAnalyzerPlugin = require('webpack-bundle-analyzer').BundleAnalyzerPlugin;
+
+var minify = {};
+
+if (process.env.NODE_ENV === 'production') {
+    minify = {
+        collapseInlineTagWhitespace: true,
+        collapseWhitespace: true
+    };
+}
+
+module.exports = {
+    entry: './src/js/main.js',
+    output: {
+        path: path.resolve(__dirname, './dist'),
+        publicPath: '/',
+        filename: '[name].[hash].js'
+    },
+    module: {
+        rules: [
+            {
+                enforce: 'pre',
+                test: /\.vue$/,
+                loader: 'eslint-loader',
+                exclude: /node_modules/
+            },
+            {
+                test: /\.vue$/,
+                loader: 'vue-loader',
+                options: {
+                    extractCSS: true
+                }
+            },
+            {
+                test: /\.js$/,
+                loader: 'babel-loader',
+                exclude: /node_modules/
+            },
+            {
+                test: /\.(png|jpg|gif|svg|ico)$/,
+                loader: 'file-loader',
+                options: {
+                    name: '[name].[hash].[ext]'
+                }
+            }
+        ]
+    },
+    plugins: [
+        new HtmlWebpackPlugin({
+            template: './src/index.ejs', // Load a custom template (ejs by default see the FAQ for details),
+            minify: minify
+        }),
+        new ExtractTextWebpackPlugin('[name].[contenthash].css')
+    ],
+    resolve: {
+        alias: {
+            'vue$': 'vue/dist/vue.esm.js'
+            // vuex: 'vue/dist/index.esm.js'
+        }
+    },
+    devServer: {
+        historyApiFallback: true,
+        noInfo: true
+    },
+    performance: {
+        hints: false
+    },
+    devtool: '#eval-source-map'
+};
+
+/*
+if (process.env.NODE_ENV === 'development') {
```

Comment:
It would be nice if we could run the analyzer like `yarn run analyze` (which launch a webpack compilation with the BundleAnalyzerPlugin included)